Carlos Gomez

crgmz opened the pull request:
Initial Draft PR for my challenge attempt.

crgmz committed:
Fix ClassDiagram.png link

crgmz committed:
Initial attempt for Passenger and TaxiDriver

errrzarrr commented on TransportCompany/TransportCompany/Car.cs:
```diff
@@ -0,0 +1,29 @@
+      {
+        Console.WriteLine($"Starting Engine for Car with {wheels} wheels.");
+      }
+      internal void SetWheels(int numOfWheels)
```
Mira el diagrama de clases para confirmar si ese es el modificador de acceso que le corresponde.

errrzarrr commented on TransportCompany/TransportCompany/Driver.cs:
```diff
@@ -4,8 +4,19 @@
+	class Driver : Person, IDriver
 	{
+		private string licenseId;
+		private Car car;
```
Driver have a car?

errrzarrr commented on TransportCompany/TransportCompany/Driver.cs:
```diff
@@ -4,8 +4,19 @@
+			car.StartEngine();
+			Console.WriteLine("I'm a driver, let's drive");
+		}
+		public void SetCar(Car car)
```
Porqué SetCar ?👀

crgmz committed:
Add Pilot, fixes in Person, Passenger, Program

Add Pilot. Add Driver license and vehicle setter/getter.
Expand TaxiDriver with payment check.
Refactor Program with multiple scenarios.
Fix for few lines Car, Person, and Passenger.

crgmz committed:
Add scenario three, Freight and FreightDriver
